rclcpp_examples/test/test_executable_linking.py.in

```diff
@@ -0,0 +1,27 @@
+# generated from rclcpp_examples/test/test_executable_linking.py.in
+
+import os
+import shlex
+import subprocess 
+
+def setup():
+    os.environ['OSPL_VERBOSITY'] = '8'  # 8 = OS_NONE
+
+
+def test_executable():
+
+    rmw_implementation = '@rmw_implementation@'
+    executables = '@RCLCPP_EXAMPLES_EXECUTABLE@'.split(';')
+    for exe in executables:
+        # Check that the executable is linked against the claimed rmw implementation
+        proc1 = subprocess.Popen(shlex.split('ldd %s'%exe), stdout=subprocess.PIPE)
```

Comment:
If you are only interested in the output of the process you can use `subprocess.check_output`. The following grep could easily be done in Python which would be preferred over calling out to `grep`.

Please use spaces around operators (e.g. `'ldd %s' % exe`).